monitoring/sysctl.go

```diff
@@ -96,6 +97,85 @@ func (c *SysctlChecker) Check(ctx context.Context, reporter health.Reporter) {
 		fmt.Sprintf("failed to query sysctl parameter %s", c.Param), trace.Wrap(err)))
 }
 
+// NewFSNRChecker creates a new checker that checks that the minimum number of file handles can be allocated
+func NewFSNRChecker(min int) health.Checker {
+	return &FSNRChecker{
+		Min: min,
+	}
+}
+
+// FSNRChecker checks the file-nr sysctl for allocatable file handles
+type FSNRChecker struct {
+	// Min is the minimum number of free descriptors before raising a health check report
+	Min int
+}
+
+// Name returns name of checker
+func (c *FSNRChecker) Name() string {
+	return FSNRCheckerID
+}
+
+// Check will load the sysctl and verify the parameter
+func (c *FSNRChecker) Check(ctx context.Context, reporter health.Reporter) {
+	content, err := Sysctl("fs.file-nr")
+	if err != nil {
+		reporter.Add(NewProbeFromErr(FSNRCheckerID, fmt.Sprintf(
+			"Failed to read sysctl fs.file-nr"), trace.Wrap(err)))
+		return
+	}
+
+	c.check(ctx, reporter, content)
+}
+
+// check will parse the provided fileNr and validate that there are enough free file handles
+func (c *FSNRChecker) check(ctx context.Context, reporter health.Reporter, fileNr string) {
+	// parsing reference
+	// https://www.kernel.org/doc/Documentation/sysctl/fs.txt
+	split := strings.Split(fileNr, "\t")
```

Comment:
nit: I haven't seen in the provided reference anywhere that the values are explicitly separated with a `TAB` - why not use a more general [strings.Fields](https://golang.org/pkg/strings/#Fields)?